psycop/common/model_training_v2/loggers/base_logger.py

```diff
@@ -1,11 +1,12 @@
-from typing import Any, Protocol
+from typing import Any, Protocol, runtime_checkable
 
 import wasabi
 
 from psycop.common.global_utils.config_utils import flatten_nested_dict
 from psycop.common.model_training_v2.metrics.base_metric import CalculatedMetric
 
 
+@runtime_checkable
```

Comment:
What does this do for us?

Reply:
Required for protocols to be used in pydantic objects; pydantic does runtime type checking, but it is limited with protocols (e.g. does not check that the type of arguments is correct).